3561/3/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  post '/comments' do
+    Comment.create(text: params[:comment],
+                   star: params[:star],
+                   place_id: params[:id],
+                   user_id: params[:user_id])
+    Place.find(params[:id]).update(rating: calculate_average_rating)
+    redirect_back
+  end
+
+  private
+
+  def calculate_average_rating
+    Comment.where(place_id: params['id']).average(:star).to_f
```

Comment:
Hmmm, looks strange, please read about associations, `@place = Place.find(params['id'])
    @comments = @place.comments`, you have this code in another place, why you can not use it here